.travis-deploy.sh

```diff
@@ -25,18 +25,13 @@ cd "$reponame"
 # Prepare gh-pages branch
 if git checkout gh-pages; then
   # Save CNAME and 404.html
-  if [ -f CNAME ]; then
-    cname="$(cat CNAME)"
-  fi
-  if [ -f 404.html ]; then
-    cp 404.html ..
-  fi
+  cp CNAME 404.html .. || :
 
   # Remove all previous files
   git ls-files | while read -r f; do git rm -rf "$f"; done
 
   # Restore 404.html
-  cp ../404.html . || :
+  cp ../404.html ../CNAME . || :
```

Comment:
The comment is outdated now and should also mention `CNAME`. Or well, be removed.

Reply:
Fixed.